### DISCUSSION TITLE
Is there a way to prevent the Table SelectColumn from shrinking?

### DISCUSSION THREAD

**Discussion by myregistration** [filamentphp/filament#13359] [Help]
### Package

Table builder

### Package Version

v3

### How can we help you?

If I have a lot of columns the select menu shrinks so you can't see the selected value until you click on the menu.  Is there a way to prevent any select menu from shrinking in general?  I was able to set the width using the extraAttributes method and style, but I'd prefer to stick with TailwindCSS and it would be better to set the overall behavior of the menu's to not shrink.  Thanks

**Comment by kslr** [comment]
same issue
![图片](https://github.com/user-attachments/assets/48325db7-7941-4772-b652-36030e32d5bb)

**Comment by howdu** [comment]
Add a min width to the table header column.

```php
Tables\Columns\SelectColumn::make('select')
    ->extraHeaderAttributes(['style' => 'min-width: 200px;']),
```
